code/modules/clothing/suits/reactive_armour.dm

```diff
@@ -425,7 +425,7 @@
 	return TRUE
 
 /obj/item/clothing/suit/armor/reactive/delimbering/proc/delimber_pulse(mob/living/carbon/human/owner, can_hit_owner = FALSE)
-	for(var/mob/living/carbon/nearby in range(range, src))
+	for(var/mob/living/carbon/nearby in range(range, get_turf(src)))
```

Comment:
```suggestion
	for(var/mob/living/carbon/nearby in range(range, owner))
```
Why check the turf instead of the person wearing the suit?

Reply:
The fire armor does it this way so if you don't want to, at least change that one as well so it's consistent.